customers/src/common/metrics/events-received.ts

```diff
@@ -0,0 +1,12 @@
+import client from 'prom-client';
+import metricsRegister from './registry';
+
+const events_received_total = new client.Counter({
```

Comment:
Metrics about messages sent and received by the service can be collected directly from MessageBroker.
I think we can leave it that way now.